AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant-model.spec.ts

```diff
@@ -89,9 +89,7 @@ describe('ParticipantModel', () => {
 
     it('should mapping return empty ParticipantModel if  JudgeResponse is null', () => {
         const judge = null;
-        console.log('here');
         participant = ParticipantModel.fromJudgeResponse(judge);
-        console.log('here');
         console.log(participant);
```

Comment:
This console.log should be removed as well. (I'm pretty sure I left these in here)